querybook/webapp/components/QueryRunButton/QueryRunButton.tsx

```diff
@@ -116,12 +118,15 @@ export const QueryEngineSelector: React.FC<IQueryEngineSelectorProps> = ({
         const queryEngineName = queryEngineById[engineId].name;
 
         return (
-            <div className="engine-selector-button flex-row">
-                <StatusIcon status={iconClass} />{' '}
-                <span>
-                    {queryEngineName}{' '}
-                    <i className="fa fa-caret-down caret-icon" />
-                </span>
+            <div className="engine-selector-button flex-center ph8">
+                <StatusIcon status={iconClass} />
+                <div className="ml4"> {queryEngineName}</div>
```

Comment:
is the extra space here intentional? If so can you use {' '} to better highlight that?

Reply:
im against using that for ui purposes